src/sentry/sentry_metrics/querying/units.py

```diff
@@ -45,6 +45,7 @@ class UnitFamily(Enum):
 
     DURATION = "duration"
     INFORMATION = "information"
+    UNKNOWN = "unknown"
```

Comment:
Added this family type, to be more explicit to the frontend when a `unit_family` is not applicable due to formula incompatibility and when the unit family it's just unknown to sentry.